HyperPlay-Gaming/hyperplay-desktop-client
# [Fix] repeat external task claims showing sync instead of claim

# Summary
This PR
- invalidates user playstreak queries on claim points, claim tasks, resync task queries
- does not show the sync button if the streak has been met even if completed_counter > 0

# Testing

1. open a g7 credits quest with 7 day streak
2. see claim all
3. click it
4. it changes to sync and streak resets to 0
5. complete streak again
6. see that the button is now claim all again
7. click it
8. it changes to sync and streak resets to 0
